### PR TITLE
Operations event moved to event core

### PR DESCRIPTION
**Pull request by kagemand-andersen**

### CODE REVIEW

File: src/main/java/org/dcsa/ovs/model/transferobjects/OperationsEventTO.java [hunk 1]
```diff
@@ -2,12 +2,12 @@
...
 import lombok.Data;
 import lombok.EqualsAndHashCode;
+import org.dcsa.core.events.model.OperationsEvent;
 import org.dcsa.core.events.model.transferobjects.PartyTO;
-import org.dcsa.ovs.model.base.AbstractOperationsEvent;
 
 @Data
 @EqualsAndHashCode(callSuper = true)
-public class OperationsEventTO extends AbstractOperationsEvent {
+public class OperationsEventTO extends OperationsEvent {
```
Comment (HenrikHL) [inline src/main/java/org/dcsa/ovs/model/transferobjects/OperationsEventTO.java:10]: Shouldn't this class also be part of EventCore?

File: src/main/java/org/dcsa/ovs/repository/OperationsEventRepository.java [hunk 2]
```diff
@@ -1,7 +1,7 @@
 package org.dcsa.ovs.repository;
```
Comment (HenrikHL) [inline src/main/java/org/dcsa/ovs/repository/OperationsEventRepository.java:1]: This file should also be part of EventCore

File: src/main/java/org/dcsa/ovs/service/OperationsEventService.java [hunk 3]
```diff
@@ -1,10 +1,7 @@
 package org.dcsa.ovs.service;
```
Comment (HenrikHL) [inline src/main/java/org/dcsa/ovs/service/OperationsEventService.java:1]: Should be part of EventCore

File: src/main/java/org/dcsa/ovs/service/OperationsEventTOService.java [hunk 4]
```diff
@@ -1,7 +1,7 @@
 package org.dcsa.ovs.service;
```
Comment (HenrikHL) [inline src/main/java/org/dcsa/ovs/service/OperationsEventTOService.java:1]: Part of EventCore

File: src/main/java/org/dcsa/ovs/service/impl/OperationsEventServiceImpl.java [hunk 5]
```diff
@@ -1,16 +1,12 @@
 package org.dcsa.ovs.service.impl;
```
Comment (HenrikHL) [inline src/main/java/org/dcsa/ovs/service/impl/OperationsEventServiceImpl.java:1]: part of EventCore

File: src/main/java/org/dcsa/ovs/service/impl/OperationsEventTOServiceImpl.java [hunk 6]
```diff
@@ -1,11 +1,10 @@
 package org.dcsa.ovs.service.impl;
```
Comment (HenrikHL) [inline src/main/java/org/dcsa/ovs/service/impl/OperationsEventTOServiceImpl.java:1]: EventCore

File: src/main/java/org/dcsa/ovs/model/transferobjects/OperationsEventTO.java (hunk 1 above)
Comment (kagemand-andersen) [reply src/main/java/org/dcsa/ovs/model/transferobjects/OperationsEventTO.java:10]: Has been moved to EventCore

**Review by HenrikHL — APPROVED** [review]

**Pull request merged**